slsDetectorServers/slsDetectorServer/src/slsDetectorServer_funcs.c

```diff
@@ -99,7 +98,12 @@ int updateModeAllowedFunction(int file_des) {
                             F_UPDATE_KERNEL,
                             F_UPDATE_DETECTOR_SERVER,
                             F_GET_UPDATE_MODE,
-                            F_SET_UPDATE_MODE};
+                            F_SET_UPDATE_MODE,
+                            F_GET_NUM_CHANNELS,
+                            F_GET_NUM_INTERFACES,
+                            F_ACTIVATE};
+    unsigned int listsize = 22;
```

Comment:
Sorry, this is in principle not related to the PR since the line didn't change. But, I couldn't help myself because it looks like a bug waiting to happen =) 

```
unsigned int listsize = 22;
```
can be replaced with
```
size_t listsize = sizeof(list)/sizeof(enum detFuncs);
```
Maybe we also want to change the name of the list variable since it technically isn't a list =) allowed_funcs?

Reply:
makes sense

Reply:
Thanks! Done! :)